HEC.FDA.Model/utilities/Serialization.cs

```diff
@@ -0,0 +1,16 @@
+﻿using H5Assist.Chunking;
+using System;
+using System.Reflection;
+
+namespace HEC.FDA.Model.utilities;
+internal class Serialization
+{
+    public static string GetXMLTagFromProperty(Type ownerType, string propertyName)
+    {
+        var oT = ownerType.GetProperty(propertyName);
+        var sP = oT.GetCustomAttribute<StoredPropertyAttribute>();
+        var name = sP.SerializedName;
```

Comment:
don't need to save Serialized name as a variable since you immediately return it. just do return sP.SerializedName

Reply:
done